.cirrus.yml

```diff
@@ -26,7 +26,7 @@ env:
   # Maximum cache period (in weeks) before forcing a new cache upload.
   CACHE_PERIOD: "2"
   # Increment the build number to force new cartopy cache upload.
-  CARTOPY_CACHE_BUILD: "0"
+  CARTOPY_CACHE_BUILD: "1"
```

Comment:
We need to nudge the build number in order to force the `cartopy_cache` miss, and a subsequent download to populate the cache.